bin/src/runRollup.js

```diff
@@ -217,7 +216,7 @@ function bundle ( options ) {
 		handleError({ code: 'MISSING_INPUT_OPTION' });
 	}
 
-	return rollup.rollup( options ).then( bundle => {
+	return rollup( options ).then( bundle => {
```

Comment:
So will `rollup.rollup(...)` be a deprecated API? Perhaps the `dist` is named rollup but is merely a module wrapper around `rollup`.

Thanks for all the knowledge. I'm new here but willing to help @Rich-Harris